doc/build-helpers/images/dockertools.section.md

```diff
@@ -767,25 +767,212 @@ $ nix-prefetch-docker --image-name mysql --image-tag 5 --final-image-name eu.gcr
 
 ## exportImage {#ssec-pkgs-dockerTools-exportImage}
 
-This function is analogous to the `docker export` command, in that it can be used to flatten a Docker image that contains multiple layers. It is in fact the result of the merge of all the layers of the image. As such, the result is suitable for being imported in Docker with `docker import`.
+This function is similar to the `docker container export` command, which means it can be used to export an image's filesystem as an uncompressed tarball archive.
+The difference is that `docker container export` is applied to containers, but `dockerTools.exportImage` applies to Docker images.
```

Comment:
Looking through the docker CLI, I see `docker image save` that seems like a better comparison:

```
$ docker image save --help
Usage:  docker image save [OPTIONS] IMAGE [IMAGE...]

Save one or more images to a tar archive (streamed to STDOUT by default)
```

```suggestion
This function is similar to the `docker image save` command, which means it can be used to export an image's filesystem as an uncompressed tarball archive.
```